server.go

```diff
@@ -2915,3 +2916,26 @@ func copyURL(u *url.URL) *url.URL {
 	*other = *u
 	return other
 }
+
+func (s *Server) StartReportingLoop(version string) chan struct{} {
+	s.reportStats(version)
+
+	ticker := time.NewTicker(24 * time.Hour)
+	for {
+		select {
+		case <-ticker.C:
+			s.reportStats(version)
+		}
+	}
+}
+
+func (s *Server) reportStats(version string) {
+	json := fmt.Sprintf(`[{"name":"reports", "columns":["os", "arch", "id", "version"], points:[["%s", "%s", "%x", "%s"]]}]`,
+		runtime.GOOS, runtime.GOARCH, s.ID(), version)
+
+	data := bytes.NewBufferString(json)
+
+	log.Printf("Reporting data: %s", json)
```

Comment:
You might like to make this a little richer, so end-users don't go "what the....?". Something like "Reporting anonymized usage metrics", or something.